doc/rtd/explanation/format.rst

```diff
@@ -159,14 +159,60 @@ a MIME archive.
 ``cloud-boothook``
 ==================
 
-This content is `boothook` data. It is stored in a file under
-:file:`/var/lib/cloud` and executed immediately. This is the earliest `hook`
-available. Note, that there is no mechanism provided for running only once. The
+This content is `boothook` data.
+This is a bash script that runs during every boot.
```

Comment:
On Debian and Ubuntu, `/bin/sh` is dash, not bash. Think we should make this statement more generic? I would assume that `#!/bin/python` and other scripting langauges are also acceptable boothooks?

Reply:
## Yes they are